operator/gitops/argocd/pipeline-service/tekton-results/watcher-cr-patch.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I see no ClusterRoleBinding for this, so why are we creating this ClusterRole ?

Reply:
This cluster role as well cluster role binding for the watcher SA comes from Tekton-results config. Why I really needed to whole cluster role is due to [this](https://github.com/openshift-pipelines/pipeline-service/pull/717/files#diff-86ff0bd69b4311eff7ccfe9ca4eefc1c4af5b2747c915c1b776b177e33c915caR31-R38). It's not in the cluster role config in Tekton Results. And adding permissions for the linked resources can't seem to be added via a Kustomize strategic merge. 
But now that I think about, it would be nice if I can create a separate CR & CRB for the tekton-results-watcher SA which will enable it to create resources for RBAC.  Are you okay with that?

Reply:
agreed. I've made the required changes.